pkgs/desktops/gnome-3/3.22/core/gnome-control-center/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
       accountsservice libkrb5 networkmanagerapplet libwacom samba libnotify libxkbfile
       shared_mime_info icu libtool docbook_xsl docbook_xsl_ns gnome3.grilo
       gdk_pixbuf gnome3.defaultIconTheme librsvg clutter clutter_gtk
-      gnome3.vino udev libcanberra_gtk3 libgudev
+      gnome3.dconf gnome3.vino udev libcanberra_gtk3 libgudev
       networkmanager modemmanager makeWrapper gnome3.gnome-bluetooth grilo tracker ];
```

Comment:
`makeWrapper` has to be replaced with `wrapGAppsHook` for control center to work.